src/numeric_filter.c

```diff
@@ -72,6 +72,9 @@ NumericFilter *NumericFilter_Parse(ArgsCursor *ac, QueryError *status) {
 }
 
 void NumericFilter_Free(NumericFilter *nf) {
+  if (!nf) {
```

Comment:
How can we get a NULL filter ?

Reply:
This fixes an actual crash that I had. I don't remember how that came to be.